src/api/app/controllers/webui/request_controller copy.rb

```diff
@@ -0,0 +1,436 @@
+class Webui::RequestController < Webui::WebuiController
```

Comment:
This isn't supposed to be in this PR, right?

Reply:
No! Sorry about that.
I pushed the correct changes already